docs/developers-guide.md

```diff
@@ -29,6 +29,11 @@ These are the set of tools which are required in order to complete any build of
 3. [Yarn package manager for Node.js](https://yarnpkg.com/) - latest release of version 1.x
 4. [Leiningen (http://leiningen.org/)](http://leiningen.org/) - latest release
 
+On a most recent stable Ubuntu/Debian, the above tools can be installed by using:
+```
+sudo apt install openjdk-11-jdk nodejs yarnpkg leiningen clojure
```

Comment:
i think (not sure) that installing `clojure` through apt gets some old abandoned clojure script that is not the current cli tool unfortunately

Reply:
yeah unfortunately.

https://packages.ubuntu.com/source/bionic/clojure

and its just a shell wrapper around rlwap
`exec $rlwrap java -cp /usr/share/java/clojure-@VER@.jar"$extra_classpath" clojure.main "$@"`

Reply:
.. and still the case with Focal (20.04): https://salsa.debian.org/clojure-team/clojure/-/blob/master/debian/scripts/clojure

Thanks for pointing this @dpsutton! I think I'll remove `clojure` from the above step. `lein run` should be sufficient for development anyway.